stats.sql

```diff
@@ -0,0 +1,21 @@
+-- -*- mode: sql; product: postgres -*-
+
+-- Create the statistics view
+-- This is what the statistics API endpoint must expose, formated as JSON.
+CREATE VIEW statistics AS
+  SELECT name, users, maxusers, data
+    FROM hosts
+    JOIN (
+            SELECT COUNT(*)
+                AS users, host
+              FROM passwd
+          GROUP BY host
+         )
+      AS count
+      ON (count.host = hosts.name)
+;
+
+
+-- Create a role for the statistics endpoint
+CREATE ROLE "statistics";
```

Comment:
This would be as the `anon` role: any sql query-er that doesn't require credentials should use it.

We don't need to allow anon connections (but that debate is unrelated)